src/outputtype.cpp

```diff
@@ -75,6 +75,8 @@ std::vector<CTxDestination> GetAllDestinationsForKey(const CPubKey& key)
 
 CTxDestination AddAndGetDestinationForScript(CKeyStore& keystore, const CScript& script, OutputType type)
 {
+    keystore.AddCScript(script);
```

Comment:
is this a bugfix?

Reply:
It moves the `AddCScript()` call from the function invoking `AddAndGetDestinationForScript()` into `AAGDFS()`; see corresponding change in src/wallet/rpcwallet.cpp